infra/nightly-resources/index.js

```diff
@@ -138,3 +138,39 @@ async function refreshLatexMacros() {
   const latexMacros = await fetch("nightlymacros.tex").then((r) => r.text());
   latexMacrosTextArea.value = latexMacros;
 }
+
+
+function addGraphs() {
+  var prevElement = document.getElementById("plots");
+  // for each plot in graphs folder, add button to show plot
+  fetch("graphs.json").then((r) => r.json()).then((data) => {
+    data.forEach((plot) => {
+      const button = document.createElement("button");
+      button.id = plot;
+      button.onclick = function () {
+        toggle(button, `\u25B6 Show ${plot}`, `\u25BC Hide ${plot}`);
```

Comment:
instead of this, can we just have a toggle at the top level div to show/hide all of the plots?

Reply:
there are already to many for that 😭